### PR TITLE
Updated s6-overlay version

### LINKED ISSUE #9: Suppress logging from s6 overlay
Worth reading this for background https://github.com/CellProfiler/docker/pull/12, specifically this
> We use s6-overlay to prevent the Linux zombie reaping problem instead of phusion-base-image since it's much more lightweight. This occurs when components of CellProfiler fail and don't get cleaned up properly (read more about it here: https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/).

The Dockerfile does this

    # Fix init and zombie process reaping problems using s6 overlay
    ADD https://github.com/just-containers/s6-overlay/releases/download/v1.11.0.1/s6-overlay-amd64.tar.gz /tmp/
    RUN gunzip -c /tmp/s6-overlay-amd64.tar.gz | tar -xf - -C /

which results in this extra logging info: 

    [fix-attrs.d] applying owners & permissions fixes...
    [fix-attrs.d] 00-runscripts: applying...
    [fix-attrs.d] 00-runscripts: exited 0.
    [fix-attrs.d] done.
    [cont-init.d] executing container initialization scripts...
    [cont-init.d] done.
    [services.d] starting services
    [services.d] done.
    ...
    [cont-finish.d] executing container finish scripts...
    [cont-finish.d] done.
    [s6-finish] syncing disks.
    [s6-finish] sending all processes the TERM signal.
    [s6-finish] sending all processes the KILL signal and exiting.
    

It will be great if this can be suppressed somehow. This might provide some info on how to do this https://github.com/just-containers/s6-overlay. 

I tried

    docker run -e S6_LOGGING=1  cellprofiler:2.2.1

But it just hangs. 

I checked by setting `S6_LOGGING=0`

    docker run -e S6_LOGGING=0  cellprofiler:2.2.1

and that behaves as expected, i.e, its output is the same as

    docker run -e  cellprofiler:2.2.1

### PR DESCRIPTION
**Pull request by shntnu**
Fixes #9

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -35,9 +35,9 @@ RUN git checkout tags/v$version
 RUN pip install --editable .
 
 # Fix init and zombie process reaping problems using s6 overlay
-ADD https://github.com/just-containers/s6-overlay/releases/download/v1.11.0.1/s6-overlay-amd64.tar.gz /tmp/
+ADD https://github.com/just-containers/s6-overlay/releases/download/v1.21.2.2/s6-overlay-amd64.tar.gz /tmp/
 
-RUN gunzip -c /tmp/s6-overlay-amd64.tar.gz | tar -xf - -C /
+RUN tar xzf /tmp/s6-overlay-amd64.tar.gz -C /
```
Comment (mcquin) [inline docker/Dockerfile]: `tar -xzf`

File: docker/README.md [hunk 2]
```diff
@@ -20,21 +12,34 @@ Dockerfile:
...
-    $ sed -i s,"ARG version=3.0.0","ARG version=${VERSION}",g Dockerfile
-    $ sed -i s,"VERSION := 3.0.0","VERSION := ${VERSION}",g Makefile
+    export VERSION=2.2.1
+    sed -i s,"ARG version=3.0.0","ARG version=${VERSION}",g Dockerfile
+    sed -i s,"VERSION := 3.0.0","VERSION := ${VERSION}",g Makefile
 
 Note: the sed commands above are for GNU sed. If you're on OS X, see [this](https://stackoverflow.com/questions/30003570/how-to-use-gnu-sed-on-mac-os-x).
 
-To push to docker hub, do the following (look up instructions at https://docs.docker.com/docker-cloud/builds/push-images/ for details)
+To build and testing docker locally:
```
Comment (mcquin) [inline docker/README.md]: testing -> test

Comment (shntnu) [reply docker/README.md]: Done

File: docker/Dockerfile (hunk 1 above)
Comment (shntnu) [reply docker/Dockerfile]: Done

**Review by mcquin — APPROVED** [review]

**Pull request merged**